style.css

```diff
@@ -114,19 +130,204 @@ body {
   width: 289px;
 }
 
-.headline ul {
-  list-style-type: none;
-  padding: 0;
-  margin: 0;
+.dark-bg ul {
   align-self: flex-start;
 }
 
-.headline ul li {
+.dark-bg ul li {
   display: inline-block;
 }
 
-.headline .squares {
+.dark-bg .squares {
   position: absolute;
   right: 0;
   bottom: 0;
 }
+
+/*  ---- Recent work ---- */
+
+.recent-work {
+  min-height: 100vh;
+  justify-items: center;
+}
+
+.title {
+  width: 85%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 130px;
+  margin-bottom: 100px;
+}
+
+.title h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 50px;
+}
+
+.title hr {
+  width: 100%;
+}
+
+.projects {
+  justify-items: center;
+  background: #fff;
+}
+
+.project {
+  width: 85%;
+  height: 386px;
```

Comment:
- [ ] Please make sure that your spacing is following the template. I kindly suggest you to remove `width:85%` from here inorder to stick to the design and design accordingly. don't forget to follow the typo(color, font) as well, as it is the core requirement of your project.
### your design👇
![recwork](https://user-images.githubusercontent.com/66992901/198508815-467f8567-18a7-4bf1-bb62-9ef4f922c913.PNG)
### figma design👇
![recefigma](https://user-images.githubusercontent.com/66992901/198508973-ac59d8ea-a394-4507-9edf-6acecb8b0b83.PNG)